src/components/card/card.stories.mdx

```diff
@@ -0,0 +1,375 @@
+import {useState} from 'react';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import Card from '.';
+import CardRow from './card-row';
+import CardFooter from './card-footer';
+import CardColumn from './card-column';
+import Heading from '../heading';
+import Link from '../link';
+import Typography from '../typography';
+import Icon from '../icon';
+import Button from '../button';
+import Box from '../box';
+import IconButton from '../icon-button';
+import VerticalDivider from '../vertical-divider';
+
+<Meta
+  title="Design System/Card"
+  parameters={{ info: { disable: true }}}
+/>
+
+## Quick start
+```js
+import Card from "carbon-react/lib/components/card";
+import CardFooter from "carbon-react/lib/components/card/card-footer";
+import CardColumn from "carbon-react/lib/components/card/card-column";
+```
+
+## Guidance, hints and tips
+- Cards: contain interactive content or controls. 
+Sometimes it might be possible to move them around on the page. They have shadows - 'card shadow' as a default, 
+and depth level 1 on hover.
+- Cards may have hover and dragging states, whereas tiles do not.
+- Neither cards nor tiles scroll.
+- The primary action area of a card is typically the card itself. 
+Often cards are one large touch target to a detail screen on a subject.
+- When an array of cards or tiles are shown together, 
+they may wrap into a grid depending on the available width and height of their container.
+
+## Default - medium spacing
+
+<Preview>
+  <Story name="default">
+    <Card>
+      <CardRow>
+        <CardColumn align="left">
+          <Heading title="Stripe - [account name]" divider={false} />
+          <Typography variant="h5">user.name@sage.com</Typography>
+        </CardColumn>
+        <CardColumn align="right">
+          <Icon type="image" />
+        </CardColumn>
+      </CardRow>
+      <CardRow>
+        <CardColumn>
+          <Typography variant="h5" weight="bold">
+              Stripe Balance
+          </Typography>
+          <Heading title="£ 0.00" divider={false} />
+          <Typography fontSize="12px">LAST ENTRY: 15 DAYS AGO</Typography>
```

Comment:
The font-weight in the `Card` design is `14px` so you don't need to override the `Typography` default here. It should be enough to use:
```
variant="p"
```
This is true for all examples that follow in this file.